DevOps/README.md

```diff
@@ -1,5 +1,34 @@
 ## DEVOPS
 
+## Version Control System - Git & Github
+
+Automate, customize, and execute your software development workflows right in your repository with GitHub Actions. You can discover, create, and share actions to perform any job you'd like, including CI/CD, and combine actions in a completely customized workflow.
+
+Visit the following resources to learn more:
+
+- [GitHub Awesome Explanation By Kunal Kushwaha ](shorturl.at/bFIT1)
+- [GitHub Actions Documentation](https://docs.github.com/en/actions)
+- [Learn GitHub Actions](https://docs.github.com/en/actions/learn-github-actions)
+- [GitHub Actions - Supercharge your GitHub Flow](https://youtu.be/cP0I9w2coGU)
+- [Automate your Workflow with GitHub Actions](https://www.youtube.com/watch?v=nyKZTKQS_EQ)
+
+
+
+## Learn a Programming Language - Python 
+
+Python is a multi-paradigm language. Being an interpreted language, code is executed as soon as it is written and the Python syntax allows for writing code in functional, procedural or object-oriented programmatic ways.  Python is frequently recommended as the first language new coders should learn, because of its focus on readability, consistency, and ease of use.  This comes with some downsides, as the language is not especially performant in most production tasks.
+
+Visit the following resources to learn more:
+
+
+- [Python Website](https://www.python.org/)
+- [Python Getting Started](https://www.python.org/about/gettingstarted/)
+- [Automate the Boring Stuff](https://automatetheboringstuff.com/)
+- [W3Schools - Python Tutorial ](https://www.w3schools.com/python/)
+- [Programmiz - Python ](https://www.programiz.com/python-programming)
+- [Codecademy - Learn Python 3](https://www.codecademy.com/learn/learn-python-3)
+
+
```

Comment:
We can exclude python since for cloud-native workloads golang is preferred for it's performance with cloud-based workloads